salt/orch/update.sls

```diff
@@ -57,9 +57,10 @@ update_modules:
   salt.function:
     - tgt: {{ master_id }}
     - name: cmd.run
-    - bg: True
     - arg:
-      - systemctl reboot
+      - sleep 15; systemctl reboot
```

Comment:
The issue was the shutdown killing the minion before it could return - I wasn't aware it could take a time parameter.

The `system.reboot` with `at_time` param may be better, but if this works, we can come back to it.